web/tasks/Wizards of Wikipedia.md

```diff
@@ -10,110 +10,168 @@ motivation: TODO.
 
 ## Dataset Description
 
-- **Homepage:**
-- **Repository:**
-- **Paper:**
-- **Point of Contact:**
+- **Homepage:** [Wizard of Wikipedia Homepage](https://parl.ai/projects/wizard_of_wikipedia/)
+- **Repository:** [Wizard of Wikipedia task in PARLAI repo](https://github.com/facebookresearch/ParlAI/tree/master/parlai/tasks/wizard_of_wikipedia)
+- **Paper:** [Wizard of Wikipedia: Knowledge-Powered Conversational agents
+](https://arxiv.org/abs/1811.01241)
+- **Point of Contact:** Option 1: Contacting via creating an issue on PARLAI [github repo](https://github.com/facebookresearch/ParlAI). Option 2: Contact authors on the above paper.
+
 
 ### Dataset and Task Summary
 
-[More Information Needed]
+Task Summary:
+The Wizard of Wikipedia is an open-domain dialogue task for training agents that can converse knowledgably about open-domain topics. In open-domain dialogue intelligent agents should exhibit the use of knowledge. This could simply be done by training models which "generate and hope" generic utterances that can be memorized in the weights of the model when mapping from input utterance(s) to output. Through grounded dialog datasets, one could rather train models that employing recalled knowledge as context. The goal here is to retrieving knowledge (relevant to context) from wikipedia, reading and conditioning on it, and finally generating natural responses.
+
+Dataset:
+The dataset consists of conversations grounded with knowledge retrieved from Wikipedia. It contains 201k utterances from 22k dialogues spanning over 1300 diverse topics, split into train, test, and valid sets. The associated Wikipedia knowledge base has 5.4M articles and 93M sentences.
 
 ### Why is this dataset part of GEM?
 
-[More Information Needed]
+Wizard-of-Wikipedia is one of the three datasets representing Dialog Generation NLG in GEM.
+
 
 ### Languages
 
-[More Information Needed]
+Contains English text only
 
 ## Meta Information
 
 ### Dataset Curators
 
-[More Information Needed]
+The dataset was curated by a team of researchers from Facebook: Emily Dinan, Stephen Roller, Kurt Shuster, Angela Fan, Michael Auli, Jason Weston.
 
 ### Licensing Information
 
-[More Information Needed]
+The [project page](https://parl.ai/projects/wizard_of_wikipedia/) does not mention a license for the dataset. ParlAI repository has [MIT License](https://github.com/facebookresearch/ParlAI/blob/master/LICENSE) 
 
 ### Citation Information
 
-[More Information Needed]
+```
+@inproceedings{dinan2019wizard,
+  author={Emily Dinan and Stephen Roller and Kurt Shuster and Angela Fan and Michael Auli and Jason Weston},
+  title={{W}izard of {W}ikipedia: Knowledge-powered Conversational Agents},
+  booktitle = {Proceedings of the International Conference on Learning Representations (ICLR)},
+  year={2019},
+}
+```
 
 ### Leaderboard
 
+Project page has a [leaderboard](https://parl.ai/projects/wizard_of_wikipedia/), though it is probably not maintained actively.
```

Comment:
I think we can remove or rephrase the second part since the "probably not maintained" bit is a bit ambiguous in what it means.